roles/wildfly_app_deploy/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Path to the application artifact (.war, .ear, .jar) on the Ansible controller
+wildfly_app_deploy_file: "/path/to/your/application.war"
```

Comment:
We usually say _deployment_ instead of _deploy_, but I think there it helps shortening the variable. I don't know if people will get confused. Thoughts?